subtrack.MAUI/Services/SubscriptionService.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.EntityFrameworkCore;
+using subtrack.DAL;
+using subtrack.DAL.Entities;
+using subtrack.MAUI.Services.Abstractions;
+
+namespace subtrack.MAUI.Services
+{
+    internal class SubscriptionService : ISubscriptionsService
+    {
+        private readonly SubtrackDbContext _context;
+        public SubscriptionService(SubtrackDbContext context) => _context = context;
+
+        public async Task<IEnumerable<Subscription>> GetSubscriptions()
+        {
+            return await _context.Set<Subscription>().ToListAsync();
```

Comment:
Personal opinion: use the property on the dbcontext instead, _context.Subscriptions